### ISSUE TITLE
Unable to access administration using custom "Limited" User Group

### ISSUE THREAD

**Issue by KingRial** [Requarks/wiki#2407]
**Describe the bug**
If you create a "maintainer" group, to allow partial access to administration page, on v2.5.126 any user owned by "Limited" user group is unable to access the administration page.

The same configuration worked on stable v2.4

**To Reproduce**
1. Go to Administration
2. Create a group with the following permissions (All permissions set except for some in administration; see screenshot below)
3. Create a user ad assign it to the just created user group
4. Login with the user credentials
5. Access Administration

**Expected behavior**
Being able to access to administration and see a dashboard with limited functions

**Screenshots**
![image](https://user-images.githubusercontent.com/886518/92585180-e805d780-f294-11ea-8b62-801102e7f671.png)

**Host Info (please complete the following information):**
 - OS: Docker - Development Environment
 - Wiki.js version: 2.5.126
 - Database engine: postgres 9-alpine

**Additional context**
I did a little of debug and from what I understood the authentication [check](https://github.com/Requarks/wiki/blob/2.5.126/server/controllers/common.js#L39) is not passing any "page" as argument.

This means the [checkAccess](https://github.com/Requarks/wiki/blob/2.5.126/server/core/auth.js#L216) method always return false even if the global permissions are fine.

To solve I just added the following code [here](https://github.com/Requarks/wiki/blob/2.5.126/server/core/auth.js#L228):

```js
// If page is not passed as argument we have enough permissions
if (page === false) {
   return true
}
```
However I am not confident enough on the project to offer a pull request with the fix without an opinion ^^

**Comment by HeavenBay** [comment]
Hi KingRial,

Thank you for sharing the solution of the problem. I still see "You are not authorized to access this resource." on the top of the page but I can edit navigation panel now with regular user! There is no error page now.

My topic is here https://github.com/Requarks/wiki/issues/2399 with the same issue.
Thanks!

**Comment by NGPixel** [comment]
Fixed by 5295e413be9d661e39b020521ff025f17933a13f
